### PR TITLE
Add env.js to store global property, add legacy mode to support rest APIs backward compatibility, and update documents 

### PR DESCRIPTION
**Pull request by hxiao608**
Signoff by: Harold Xiao haoruxia@amazon.com

**Is your feature request related to a problem? Please provide an existing Issue # , or describe.**

**Describe the solution you are proposing**
Add a new js file for setting and getting global properties, introduce a legacy flag to enable perfTop to run in a legacy mode, which will send out API request to opendistro endpoint 

**Describe alternatives you've considered**

**Additional context**

**Test**
Local build pass and succesfully test the new endpoint with the sample metrics
command: ``./build/opensearch-perf-top-macos --dashboard ${Dashboard} --endpoint localhost:9600``
results: 
Screen Shot 2021-05-21 at 4 45 12 PM
Screen Shot 2021-05-21 at 4 52 52 PM

### Check List
- [x] New functionality includes testing.
    - [x] All tests pass
- [ ] New functionality has been documented.
    - [ ] New functionality has javadoc added
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/perftop/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Review by jotok — APPROVED** [review]

**Comment by sruti1312** [comment]
Since we dont have automated testing. Can we manually test the artifact and confirm everything is working as expected? 
Please add the manual testing details as well to the PR. Thanks!

**Comment by hxiao608** [comment]
Perftop can now render metrics with the new ``_plugins`` endpoint. The legacy mode will be able to render metrics after the setting up the Rest API BC for original ``_opendistro`` endpoint .

File: lib/env.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const BASE_URL = "/_plugins/_performanceanalyzer";
+const BASE_URL_PA = BASE_URL + "/metrics";
```
Comment (sruti1312) [inline lib/env.js:2]: Can we rename this to BASE_URL_METRICS?

**Review by sruti1312 — APPROVED** [review]

File: lib/env.js (hunk 1 above)
Comment (hxiao608) [reply lib/env.js]: Done

**Pull request merged**